### ISSUE TITLE
Buildbot upgrade-master keeps showing up

### ISSUE THREAD

**Issue by jessielaf** [buildbot/buildbot#4805]
Hi guys,

I am trying to start buildbot in my vagrant machine which is running ubuntu/bionic64 on my windows 10 machine.

I followed http://docs.buildbot.net/latest/tutorial/firstrun.html but when running `buildbot start master` I get
"`buildbotNetUsageData` is not configured and defaults 
to basic"

So I add `c['buildbotNetUsageData'] = None` to the master.cfg and restart the master by running `buildbot restart master`. Now the first error is gone but I get 
```
2019-06-03 13:15:12+0000 [-] The Buildmaster database needs to be upgraded before this version of     
2019-06-03 13:15:12+0000 [-] buildbot can run.  Use the following command-line
2019-06-03 13:15:12+0000 [-]
2019-06-03 13:15:12+0000 [-]     buildbot upgrade-master /vagrant/master
```

So I run 
```
buildbot upgrade-master /vagrant/master
```

Which returns
```
checking basedir
checking for running master
checking master.cfg
upgrading basedir
creating /vagrant/master/master.cfg.sample
upgrading database (sqlite:/state.sqlite)
upgrade complete
```

Now I restart the buildbot by running `buildbot restart master`

But I get the `upgrade-master` error again. No matter how often I upgrade master the error keeps showing up

**Comment by p12tic** [comment]
Could you give more information about how you acquired buildbot? Which version of buildbot is it, how you configured the master first time and so on?

**Comment by jessielaf** [comment]
I use the latest Buildbot which is 2.3.1. I used the exact commands described on http://docs.buildbot.net/latest/tutorial/firstrun.html so:
```
cd /vagrant
pip install 'buildbot[bundle]'
buildbot create-master master
mv master/master.cfg.sample master/master.cfg
```

**Comment by p12tic** [comment]
I could not reproduce what you're seeing. Could it be that the database itself has been somehow corrupted in the process? Could you remove the `master` directory created by the `create-master` command, redo the commands and see whether the issue is still there?

**Comment by jessielaf** [comment]
I also tried it with mysql and tried to delete the master folder both gave me the same result as above. I also added my Vagrantfile for more clarity.

**Comment by p12tic** [comment]
Interesting. Could you attach the state.sqlite file from the master directory to this bug report? Maybe I can reproduce the issue with it.

**Comment by jessielaf** [comment]
Yes thanks for looking into this! I added it as a .zip because github does not support .sqlite files.

[state.zip](https://github.com/buildbot/buildbot/files/3280741/state.zip)
